playground/src/editor.tsx

```diff
@@ -129,6 +144,102 @@ export function Editor(props: {
       lineNumbersMinChars: 3,
     });
 
+    monaco.languages.registerCompletionItemProvider("qsharp", {
```

Comment:
I'm not sure this is the right location to register services for languages with monaco. These should only be run once. This useEffect will run every time this component is created (e.g. if you navigate to Katas and back, this will run several times).

Reply:
(Also, for a page like the Katas where we can have multiple Monaco editors, this will run multiple times for the one page (once for each editor). So even if you unregistered on component tear-down, it'd still run multiple times for the page).

Reply:
That makes sense. Moving to `App` initialization (same spot we create the compiler & LS).